experimental/traffic-portal/.eslintrc.json

```diff
@@ -37,10 +37,16 @@
 				"createDefaultProgram": true
 			},
 			"extends": [
-				"plugin:@angular-eslint/ng-cli-compat",
-				"plugin:@angular-eslint/ng-cli-compat--formatting-add-on",
+				"eslint:recommended",
+				"plugin:@typescript-eslint/recommended",
+				"plugin:@angular-eslint/recommended",
```

Comment:
This looks like it's adding at least `eslint:recommended`, not sure if the other two are just new names for the ng-cli-compat sets. But adding new linter rules seems out-of-scope for this PR.